usmqe/web/tendrl/import_cluster.py

```diff
@@ -10,30 +10,28 @@
 from usmqe.web.tendrl.mainpage.navpage.pages import NavMenuBars
 from usmqe.web.tendrl.mainpage.clusters.cluster_list.pages import\
     ClustersList, check_hosts
-from usmqe.web.tendrl.mainpage.tasks.pages import TaskDetails
 from usmqe.web.tendrl.mainpage.clusters.cluster.pages import ClusterMenu
 from usmqe.web.tendrl.task_wait import task_wait
 
 
 IMPORT_TIMEOUT = 3600
 
 
-def import_cluster_wait(driver, import_task_details):
```

Comment:
Because the method will need not require just `TaskDetails` instance, but also `TaskEvents` instance. And I don't like the idea to give both to the method when the method could easily creates them. Moreover the second one does not exist from beginning.